### PR TITLE
Hotfix/7.2.2

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
-version=7.2.1
-iexecCommonVersion=6.1.0
+version=7.2.2-rc2
```
Comment (mcornaton) [inline gradle.properties:1]: Do not forget before merging :)
```suggestion
version=7.2.2
```

**Review by mcornaton — APPROVED** [review]
Thanks!

**Review by jeremyjams — COMMENTED** [review]
Can you please add new changelog section?

File: gradle.properties (hunk 1 above)
Comment (jbern0rd) [reply gradle.properties]: done in a43eb67

**Comment by jbern0rd** [comment]
> Can you please add new changelog section?

Done in 7177882

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[7.2.2]](https://github.com/iExecBlockchainComputing/iexec-core/releases/tag/v7.2.2) 2022-12-20
+
+* Use `iexec-common` version [6.2.0](https://github.com/iExecBlockchainComputing/iexec-common/releases/tag/v6.2.0).
+* Use `okhttp` version 4.9.0 to keep it in sync with web3j dependency updated in `iexec-common`.
```
Comment (jeremyjams) [inline CHANGELOG.md:8]:
```suggestion
* Use `okhttp` version 4.9.0 to keep it consistent with the one in the web3j dependency imported by `iexec-common`.
```

**Review by jeremyjams — APPROVED** [review]
Thanks :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (jbern0rd) [reply CHANGELOG.md]: done in 5fc0509

**Pull request merged**